microcosm_flask/encryption/conventions/crud_adapter.py

```diff
@@ -0,0 +1,41 @@
+from microcosm_flask.conventions.crud_adapter import CRUDStoreAdapter
+
+
+class EncryptableCRUDStoreAdapter(CRUDStoreAdapter):
+    """
+    Adapt the CRUD conventions callbacks to the `EncryptableStore` interface.
+
+    """
+
+    def update(self, **kwargs):
+        """
+        Support re-encryption by enforcing that every update triggers a
+        new encryption call, even if the the original call does not update
+        the encrypted field.
+
+        """
+        encrypted_field_name = self.store.model_class.__plaintext__
+
+        id_ = kwargs[self.identifier_key]
+        current_model = self.store.retrieve(id_)
+        current_value = current_model.plaintext
+
+        null_update = (
+            # Check if the update is for the encrypted field, and if it's explicitly set to null
```

Comment:
That's going to be problematic  - with default values:
```
update(self, foo=None, **kwargs):
    return super().update(foo=foo, **kwargs)
```

Reply:
I'd argue that updates shouldn't ever pass in default values. You should only be specifying exactly what you want to update here.